caf-logging-log4j2/src/test/java/com/github/cafapi/logging/log4j2/LogInjectionTests.java

```diff
@@ -56,6 +58,11 @@ public void testLogMessageWithNewline() throws Exception
         final String log = utf8Decode(logStream);
         final long logLineCount = countLines(log);
 
+        System.out.println("log " + log);
+        String sd = "\n";
+        IntStream df = sd.chars();
+        System.out.println("lb " + df.findFirst().getAsInt());
```

Comment:
Why would adding Serilog support involve making any changes to the Log4J2 tests?

Reply:
Quick test, I removed it